bindings/python/core_objects/include/py_core_objects/py_variant_extractor.h

```diff
@@ -169,6 +170,7 @@ template <typename DaqType,
           typename Variant>
 daq::ObjectPtr<std::remove_pointer_t<DaqType>> getVariantValue(Variant& v)
 {
+    py::gil_scoped_acquire acquire;
```

Comment:
As discussed, this should be investigated why this line is neccessary.

Reply:
It's acquiring GIL only for python types now